src/api/data/cache/charthelper/chart_package_helper.go

```diff
@@ -134,6 +136,19 @@ var ensureChartDataDir = func(chart *models.ChartPackage) error {
 	return nil
 }
 
+var cleanChartDataDir = func(chart *models.ChartPackage) error {
+	dir := chartDataDir(chart)
+	if _, err := os.Stat(dir); err != nil {
```

Comment:
why do you want to return an error if the dir doesn't exist?

i.e. I wonder why cannot this just be:

```
var cleanChartDataDir = func(chart *models.ChartPackage) error {
  return os.RemoveAll(chartDataDir(chart))
}
```